packages/apputils/src/toolbar.tsx

```diff
@@ -344,71 +360,89 @@ namespace Toolbar {
  * A widget which acts as a button in a toolbar.
  */
 export
-class ToolbarButton extends Widget {
+class ToolbarButton extends VDomRenderer<null> {
   /**
    * Construct a new toolbar button.
    */
-  constructor(options: ToolbarButton.IOptions = {}) {
-    super({ node: document.createElement('button') });
-    Styling.styleNodeByTag(this.node, 'button');
-    this.addClass(TOOLBAR_BUTTON_CLASS);
-    this._onClick = options.onClick || Private.noOp;
-
-    const classes = options.className ?
-      options.className.trim().replace(/\s{2,}/g, ' ').split(/\s/) : null;
+  constructor(options: ToolbarButton.IOptions) {
+    super();
+    this.props = options;
+    this.addClass(TOOLBAR_ITEM_CLASS);
+    this.removeClass('p-Widget');
```

Comment:
Just curious, why do we have to remove this class?

Reply:
With the class: 

![image](https://user-images.githubusercontent.com/512354/40083774-0eaa1028-584a-11e8-8462-2b5cdb085280.png)

Without the class:

![image](https://user-images.githubusercontent.com/512354/40083811-2269f27c-584a-11e8-9c8c-f404c5c6c8f7.png)